src/main/kotlin/inc/evil/config/Databases.kt

```diff
@@ -37,4 +40,293 @@ object DatabaseSingleton {
+                    classroom = Classroom.findById(UUID.fromString("your-classroom-id-here"))!!
+                    date = LocalDate.parse("2024-06-22")
+                    startTime = LocalTime.parse("15:00")
+                    endTime = LocalTime.parse("18:00")
+                    type = ReservationType.EVENT
+                },
+                Reservation.new {
+                    title = "Database Systems Lecture"
+                    user = User.findById(UUID.fromString("your-user-id-here"))!!
+                    classroom = Classroom.findById(UUID.fromString("your-classroom-id-here"))!!
+                    date = LocalDate.parse("2024-06-23")
+                    startTime = LocalTime.parse("08:00")
+                    endTime = LocalTime.parse("10:00")
+                    type = ReservationType.LECTURE
+                },
+                Reservation.new {
+                    title = "Art Workshop"
+                    user = User.findById(UUID.fromString("your-user-id-here"))!!
+                    classroom = Classroom.findById(UUID.fromString("your-classroom-id-here"))!!
+                    date = LocalDate.parse("2024-06-26")
+                    startTime = LocalTime.parse("14:00")
+                    endTime = LocalTime.parse("16:00")
+                    type = ReservationType.STUDENTS_CLUB_MEETING
+                },
+                Reservation.new {
+                    title = "Mobile App Development Test"
+                    user = User.findById(UUID.fromString("your-user-id-here"))!!
+                    classroom = Classroom.findById(UUID.fromString("your-classroom-id-here"))!!
+                    date = LocalDate.parse("2024-06-27")
+                    startTime = LocalTime.parse("11:00")
+                    endTime = LocalTime.parse("13:00")
+                    type = ReservationType.TEST
+                },
+                Reservation.new {
+                    title = "Photography Workshop"
+                    user = User.findById(UUID.fromString("your-user-id-here"))!!
+                    classroom = Classroom.findById(UUID.fromString("your-classroom-id-here"))!!
+                    date = LocalDate.parse("2024-06-28")
+                    startTime = LocalTime.parse("15:00")
+                    endTime = LocalTime.parse("17:00")
+                    type = ReservationType.STUDENTS_CLUB_MEETING
+                },
+                Reservation.new {
+                    title = "Machine Learning Lecture"
+                    user = User.findById(UUID.fromString("your-user-id-here"))!!
+                    classroom = Classroom.findById(UUID.fromString("your-classroom-id-here"))!!
+                    date = LocalDate.parse("2024-06-26")
+                    startTime = LocalTime.parse("09:00")
+                    endTime = LocalTime.parse("11:00")
+                    type = ReservationType.LECTURE
+                }
+            )
+        }
+    }
```

Comment:
Tutaj nie za bardzo rozumiem to "your_userid", bo chodzi o to, żeby uruchomić ten kod i żeby zawsze mieć poprawne dane bez edytowania pliku. Jeśli zawsze dodajemy n Userów, to można np. użyć User.all().toList().get(x) gdzie 0 < x < n.

Chyba, że masz inny pomysł. Trzeba pamiętać, że UUID generuje tutaj baza danych.